presto-accumulo/pom.xml

```diff
@@ -0,0 +1,376 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.facebook.presto</groupId>
+        <artifactId>presto-root</artifactId>
+        <version>0.152-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>presto-accumulo</artifactId>
+    <description>Presto - Accumulo Connector</description>
+    <packaging>presto-plugin</packaging>
+
+    <properties>
+        <accumulo.version>1.6.2</accumulo.version>
+        <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+        <air.test.fork-mode>always</air.test.fork-mode>
+        <curator.version>2.9.1</curator.version>
+        <jackson.version>2.4.4</jackson.version>
+        <junit.version>4.12</junit.version>
+        <mdep.analyze.skip>true</mdep.analyze.skip>
+    </properties>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <configuration>
+                            <!-- ZooKeeper is incorrectly flagged as unused -->
+                            <!-- And we need to include the hadoop sub-dependency to exclude transitive dependencies -->
+                            <ignoredUnusedDeclaredDependencies>
+                                <ignoredUnusedDeclaredDependency>org.apache.hadoop:hadoop-auth</ignoredUnusedDeclaredDependency>
+                                <ignoredUnusedDeclaredDependency>org.apache.hadoop:hadoop-client</ignoredUnusedDeclaredDependency>
+                                <ignoredUnusedDeclaredDependency>org.apache.hadoop:hadoop-hdfs</ignoredUnusedDeclaredDependency>
+                                <ignoredUnusedDeclaredDependency>org.apache.hadoop:hadoop-minicluster</ignoredUnusedDeclaredDependency>
+                                <ignoredUnusedDeclaredDependency>org.apache.zookeeper:zookeeper</ignoredUnusedDeclaredDependency>
+                            </ignoredUnusedDeclaredDependencies>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>com.ning.maven.plugins</groupId>
+                <artifactId>maven-dependency-versions-check-plugin</artifactId>
+                <configuration>
+                    <exceptions>
+                        <exception>
```

Comment:
I receive the below error removing the plugin configuration, but I've excluded the offending transitive dependencies from the various hadoop artifacts.